### PR TITLE
A0-2356: Prepare ink-wrapper-types for release to crates

### PR DESCRIPTION
**Pull request by obrok**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,10 @@ let sub_contract_events = all_events.for_contract(sub_contract);
 The `all_events` object above may contain events from multiple contracts if the contract called into them. In that case,
 you can filter and parse these events by calling `for_contract` on it, with the various contracts you're interested in.
 
+### Example
+
+Look at `test-project` in the project's repo for a fuller example.
```
Comment (deuszx) [inline README.md:115]: I think we're lacking (an entry in) README where we explain how to generate the missing files (`psp22_contract.rs` and `test_contract.rs`).

We can also fix [this section](https://github.com/Cardinal-Cryptography/ink-wrapper/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R43-R57) and use the actual version numbres. I think it's safe to use `0.1.0` for `ink-wrapper-types`, right?

Comment (obrok) [reply README.md:115]: > I think we're lacking (an entry in) README where we explain how to generate the missing files (psp22_contract.rs and test_contract.rs).

I'll add that.

> We can also fix [this section](https://github.com/Cardinal-Cryptography/ink-wrapper/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R43-R57) and use the actual version numbres. I think it's safe to use 0.1.0 for ink-wrapper-types, right?

I wanted to update this once I actually do the release. Should we give separate version numbers to `ink-wrapper-types` and `ink-wrapper`? Or should they be synchronized?

Comment (deuszx) [reply README.md]: We can start with using synchronized versioning and move to something more sophisticated if it turns out to be needed.

Comment (obrok) [reply README.md]: Ok, then I'll set `ink-wrapper-types` to `0.2.0` when we do the release for both of them.

**Review by deuszx — APPROVED** [review]

**Pull request merged**